### DISCUSSION TITLE
Object of type int32 is not JSON serializable

### DISCUSSION THREAD

**Discussion by jessssschiu** [Tunny-gh/Tunny_in_Open_Sea#147] [General]
Hi!

I tried to run the gh. file (tunny_constraint.gh) I downloaded here: https://discourse.mcneel.com/t/how-to-choose-between-the-multi-objective-optimization-plugins/148633/5

but got this runtime error message: "Object of type int32 is not JSON serializable"

I am using Tunny v 0.6.0.

Help would be appreciated!

**Comment by hrntsm** [comment]
Hi @jessssschiu 

Thanks for the post. I will check it out!
Which component outputs this error message?

**Reply by jessssschiu** [reply]
Hi! I upgraded Rhino and ran the same script. It is working now :)
